xstream/server/server/rpc.go

```diff
@@ -116,7 +116,10 @@ func (t *Server) StopRule(name string, reply *string) error {
 	*reply = stopRule(name)
 	return nil
 }
-
+func (t *Server) RecoverRule(name string, reply *string) error {
```

Comment:
This seems no effect. The caller can call the recoverRule directly. And this is not an exported RPC API. I suggest this function can be removed.